modules/agent-installer-architectures.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="agent-install-networking_{context}"]
+= Optional: Verifying the architecture of an Agent-based installer cluster
+
+After installing an {product-title} cluster using the Agent-based Installer, you can check the architecture of the cluster.
+
+.Prerequisites: 
+
+* You installed the {oc-first}.
+* You have the `openshift-install` binary.
```

Comment:
```suggestion
* You have the `openshift-install` binary.
```
What does this mean to "have" the binary? Should it be installed?

Reply:
I just checked we call `openshift-install` binary as `installation program` in other chapter. we can keep here consitant with that. https://docs.openshift.com/container-platform/4.15/installing/installing_aws/installing-aws-default.html#installation-obtaining-installer_installing-aws-default
```suggestion
* You have downloaded the installation program.
```

Reply:
Updated as per the suggestion.